README.md

```diff
@@ -5,35 +5,27 @@
 
 Static server that hosts [Firefox Account sign up](https://accounts.firefox.com), sign in, email verification, etc. flows.
 
+## Quick Start
 
-Follow the instructions on:
-	[fxa-local-dev](https://github.com/mozilla/fxa-local-dev) to get a local version running.
-	
-The above link helps to install the following pre-requisites, and set up a development environment.
+Clone the repository, make sure you have [required prerequisites](https://github.com/mozilla/fxa-local-dev#dependencies) installed.
+Run `npm install` and `npm run start-remote`.
+This will start a local fxa-content-server on [http://127.0.0.1:3030](http://127.0.0.1:3030) that works with remote Firefox Accounts servers.
```

Comment:
Yeah good point, I'll open a follow up issue because we need to extract the Firefox launcher from fxa-local-dev first